lib/include/openamp/remoteproc.h

```diff
@@ -410,6 +414,9 @@ struct remoteproc_ops {
 	int (*stop)(struct remoteproc *rproc);
 	int (*shutdown)(struct remoteproc *rproc);
 	int (*notify)(struct remoteproc *rproc, uint32_t id);
+	int (*get_mem)(struct remoteproc *rproc, const char *name,
+		       metal_phys_addr_t pa, metal_phys_addr_t da,
+		       void *va, size_t size, struct remoteproc_mem *mem);
```

Comment:
>> If you want to propose this kind of API i think you also need to implement this for get_mem ops to allow a porting layer relying on a list. With an appropriate description of the ops that describes the expected behavior.

>Done.
my point was also about get_mem itself
```
 -	int (*get_mem)(struct remoteproc *rproc, const char *name,
 +	struct remoteproc_mem *(*get_mem)(struct remoteproc *rproc, const char *name,
		       metal_phys_addr_t pa, metal_phys_addr_t da,
		       void *va, size_t size, struct remoteproc_mem *mem);

```
This would allow the use case where get_mem returns its own pointer.

Reply:
Yes, the implementation could ignore mem argument and return the internal managed remoteproc_mem directly, but in this case it's more simpler to call remoteproc_add_mem in the initialize phase and reuse the common code instead.

Reply:
Done.